valhalla/thor/pathalgorithm.h

```diff
@@ -32,8 +32,10 @@ class PathAlgorithm {
   /**
    * Constructor
    */
-  PathAlgorithm()
-      : interrupt(nullptr), has_ferry_(false), not_thru_pruning_(true), expansion_callback_() {
+  PathAlgorithm(uint32_t max_reserved_labels_count, bool clear_reserved_memory)
+      : interrupt(nullptr), has_ferry_(false), not_thru_pruning_(true), expansion_callback_(),
+        max_reserved_labels_count_(std::move(max_reserved_labels_count)),
```

Comment:
`std::move` - what for ? integer types don't have move-constructors, it will be always copying